Tests/Fixtures/MatthiasDependencyInjectionTestExtension.php

```diff
@@ -5,11 +5,20 @@
 use Symfony\Component\DependencyInjection\Definition;
 use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
 use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
 use Symfony\Component\Config\FileLocator;
 
-class MatthiasDependencyInjectionTestExtension implements ExtensionInterface
+class MatthiasDependencyInjectionTestExtension implements ExtensionInterface, PrependExtensionInterface
```

Comment:
I agree, it may seem like a lot of fuss for nothing extra, but it allows you to see each case (with or without extension interface) in plain sight, basically untangling features.